packages/compose/src/hooks/use-copy-to-clipboard/index.js

```diff
@@ -37,11 +44,11 @@ export default function useCopyToClipboard( text, onSuccess ) {
 	const onSuccesRef = useUpdatedRef( onSuccess );
```

Comment:
Nit (and arguably out of scope), but found this typo

```suggestion
	const onSuccessRef = useUpdatedRef( onSuccess );
```